spk/jellyfin/Makefile

```diff
@@ -8,13 +8,14 @@ DSM_UI_DIR = app
 
 DEPENDS = cross/libstdc++ cross/jellyfin cross/jellyfin-web
 
-UNSUPPORTED_ARCHS = $(PPC_ARCHS) $(ARMv5_ARCHS) $(ARMv7_ARCHS) $(ARMv7L_ARCHS) $(i686_ARCHS)
+# x64 and armv8 archs are supported only.
+UNSUPPORTED_ARCHS = $(32bit_ARCHS)
 
 MAINTAINER = stevenliuit
 DESCRIPTION = "The Free Software Media System. It is an alternative to the proprietary Emby and Plex."
 DISPLAY_NAME = Jellyfin
 STARTABLE = yes
-CHANGELOG = "1. Update jellyfin to 10.8.9<br>2. Update dotnet 6.0.405"
+CHANGELOG = "1. Update jellyfin to 10.8.9<br/>2. Update dotnet to 6.0.13<br/>3. Remove support for 32-bit archs."
```

Comment:
I saw a similar modification from @publicarray regarding `<br/>`. If you can, I'd really like your help validating the need for this because I have two bits of information which conflict with this:

1. The example for the `CHANGELOG` in our own Wiki page on [Available Variables in spk/ Makefiles](https://github.com/SynoCommunity/spksrc/wiki/Makefile-variables#available-variables-in-spk-makefiles) shows `<br>` separating multiple entries
2. The W3C Markup Validation Service has some documentation on [Trailing slashes](https://github.com/validator/validator/wiki/Markup-%C2%BB-Void-elements#trailing-slashes) which indicates that "_a trailing slash to mark start tags as self-closing_" ... "_does not concur with what the HTML standard states_"

As such, I'd really like some clarity on the correct use of HTML with void elements in the change log.

Reply:
Reading the docs the trailing slash is only a problem for tags with attributes. 
So I leaf it as is (and updated the wiki page)

Reply:
Thanks for the response. I'll correct all my pending PRs based on this.